content/en/platform/corda/5.0/deploying-operating/observability/tracing.md

```diff
@@ -0,0 +1,31 @@
+---
+date: '2023-05-10'
+version: 'Corda 5.0 Beta 4'
+title: "Tracing Framework"
+menu:
+  corda5:
+    parent: corda5-cluster-observability
+    identifier: corda5-cluster-tracing
+    weight: 4000
+section_menu: corda5
+---
+
+# Tracing Framework
+
+As a Corda Cluster Operator, you may want to configure sample rate when deploying the Helm chart. You can do this using the
+`--trace-samples-per-second` command line option that:
+
+* defaults to 1 sample per second if not set
+* allows specifying unsigned numeric values as samples per second
+* allows `unlimited` value to to sample all requests
+
+The tracing configuration supported by the Helm chart contains the following information in the `values.yaml` file:
```

Comment:
I'm told we haven't documented worker command line parameters. I think we should not document this one alone. Lets remove the command line parameter and focus on the Helm chart.

```suggestion
As a Corda Cluster Operator, you may want to configure a tracing server and sample rate when deploying the Helm chart. The tracing configuration supported by the Helm chart contains the following configuration in the `values.yaml` file:
```